doc/source/user/building.rst

```diff
@@ -94,10 +94,23 @@ Testing
 -------
 
 Make sure to test your builds. To ensure everything stays in shape, see if
-all tests pass::
+all tests pass
+
+First install all the python dependencies that will be required for testing 
+by running the below command in the root directory::
```

Comment:
```suggestion
all tests pass.

First install all the Python dependencies that will be required for testing 
by running the below command in the root directory::
```

IMO this recommendation could be simplified further with something like:

```rst
The test suite requires additional dependencies which can easily be installed with::

    python -m pip install -r test_requirements.txt
```

The grammatical fixes are necessary, but feel free to ignore the wording suggestion.

Reply:
Got it. I am open to keeping the wording suggestion suggested here, I will surely check what comes in the next commit to be grammatically correct.


The only thing that I found on the official docs [Python Interpreter](https://docs.python.org/3/tutorial/interpreter.html) is that `-m` is used to run a module as a script. 

Just one query is `python -m pip install -r test_requirements.txt` a safer option or `pip install -r test_requirements.txt` is also correct?